test/monster_test/monster_test.c

```diff
@@ -5,6 +5,7 @@
 
 #include "flatcc/support/hexdump.h"
 #include "flatcc/support/elapsed.h"
+#include "../../config/config.h"
```

Comment:
I already tried "config.h" but no luck. I could look into cmakelists for monster_test and try to add it to include path, so that ../.. is not needed - I don't like current solution either, just wasn't unsure if modifying build parameters here is acceptable, given other places with ../..

As for rsync - I'll add them in couple of minutes